packages/react-composites/src/composites/CallComposite/components/CallReadinessModal.tsx

```diff
@@ -35,13 +35,15 @@ export const CallReadinessModal = (props: {
     camera: PermissionState;
     microphone: PermissionState;
   }) => void;
+  cameraPermissionGranted: boolean | undefined;
```

Comment:
we have two sets of props here that look the same `videoState` and `cameraPermissionGranted` -- what's the difference between these two?